Integrations/integration-Symantec_Site_Review.yml

```diff
@@ -0,0 +1,205 @@
+description: Symantec Site Review (Bluecoat)
+configuration:
+- display: API Server URL
+  name: url
+  defaultvalue: https://sitereview.bluecoat.com
+  type: 0
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Trust any certificate (unsecure)
+  name: unsecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |-
+    import requests
+    import json
+
+    requests.packages.urllib3.disable_warnings()
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    """
+    GLOBAL VARIABLES
+    """
+
+    SERVER = demisto.params()['url'][:-1] if demisto.params()['url'].endswith('/') else demisto.params()['url']
+    BASE_URL = SERVER
+    VERIFY_CERTIFICATE = not demisto.params().get('unsecure', False)
+
+    """
+    Helper Functions
+    """
+    def return_error_and_exit(message):
+        demisto.results({
+            'Type': entryTypes['error'], # note, warning, error
+            'ContentsFormat': formats['text'], # json, markdown, file, text
+            'Contents': message
+        })
+        sys.exit(0)
+
+    def http_request(method, url, body=None):
+
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+
+        res = requests.request(method, url, verify=VERIFY_CERTIFICATE, headers=headers, data=body)
+
+        if res.status_code < 200 or res.status_code >= 300:
+            return_error_and_exit('Failed to retrieve data.\nURL: {}, Status Code: {}, Response: {}'.format(url, res.status_code, res.text))
+
+        return res.json()
+
+    """
+    Symantec Site Review Commands
+    """
+
+    def list_risk_levels():
+        full_url = BASE_URL + '/resource/risklevels'
+        res = http_request('GET', full_url)
+        return res
+
+    def list_risk_levels_command():
+        raw_list = list_risk_levels()
+        #demisto.log(raw_list)
+        risk_content = []
+        for level in raw_list:
+            content = {
+                "Risk Level": level['number'],
+                "Description": level['name']
+            }
+            risk_content.append(content)
```

Comment:
create another dictionary for the context using the camelize function explained in the other function